### PR TITLE
1736157933 monterey upgrade

### PR DESCRIPTION
**Pull request by kirasglimmer**
support Monterey

replaces https://github.com/daptiv/strap/pull/39

### CODE REVIEW

**Review by jaredreynolds — APPROVED** [review]
LGTM

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          docker build --tag strap .
+          docker tag strap mikemcquaid/strap:master
+          docker tag strap ghcr.io/mikemcquaid/strap:master
+          docker tag strap registry.heroku.com/macos-strap/web
+
+      - name: Deploy the Docker image to GitHub Packages
+        if: github.ref == 'refs/heads/master'
+        run: |
+          echo ${{secrets.GITHUB_TOKEN}} | \
+            docker login --username=mikemcquaid --password-stdin ghcr.io
```
Comment (greglboxer) [inline .github/workflows/tests.yml:95]: not sure we want to be copying this docker action from the main strap repo.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@master
+        with:
+          bundler-cache: true
+
+      - run: script/style
+
+  tests:
+    runs-on: macos-12
```
Comment (greglboxer) [inline .github/workflows/tests.yml:27]: github actions support being run on a mac? neat.

**Review by greglboxer — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (kirasglimmer) [reply .github/workflows/tests.yml]: deleted

File: .github/workflows/tests.yml (hunk 2 above)
Comment (kirasglimmer) [reply .github/workflows/tests.yml]: deleted the GitHub actions

**Review by greglboxer — APPROVED** [review]

**Pull request merged**